### ISSUE TITLE
How do I find out which domains are on which droplet?

### ISSUE THREAD

**Issue by opn** [digitalocean/api-v2#131]
I cannot see how to do this with the v2 API? Lot's of info about all the domains across all the droplets, and lots of info about droplets - but nothing that ties them together?

The only way i can think of doing it is by extracting IP address information from both sets of API calls.

**Comment by phillbaker** [comment]
@opn that's correct, at this point there isn't a direct look up between the two objects. 

What's the problem you're trying to solve?
